docker-compose.yml

```
@@ -0,0 +1,72 @@
+services:
+  db:
+    image: postgres:16-alpine
+    container_name: postgres_lms
+    env_file:
+      - .env
+    volumes:
+      - postgres-data:/var/lib/postgresql/data
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -U $POSTGRES_USER" ]
```

В хэлфчеке необходимо также прописывать название базы
```suggestion
      test: [ "CMD-SHELL", "pg_isready -U $POSTGRES_USER -d $POSTGRES_DB" ]
```